ptvsd/wrapper.py

```diff
@@ -1455,15 +1455,33 @@ def on_ptvsd_systemInfo(self, request, args):
             except AttributeError:
                 impl_desc = 'Python'
 
+        def version_str(v):
+            return '{}.{}.{}{}{}'.format(
+                v.major,
+                v.minor,
+                v.micro,
+                v.releaselevel,
+                v.serial)
+
+        try:
+            impl_name = sys.implementation.name
+        except AttributeError:
+            impl_name = 'Python'
```

Comment:
Can you change this to return None on exception, like and the same for `impl_desc` above and `impl_version` below? Basically so that the client knows whether this information was successfully obtained, or not. Falling back to "Python" is old debugger code, where it was done for ease of implementation mostly; but I think it's best for debugger to just report values as it sees them, and for the IDE to do whatever formatting and missing value substitution that's necessary to represent that data to the user.

Reply:
will do